### PR TITLE
Several smaller matches in 1BCF0 and 5BF20

### PR DESCRIPTION
**Pull request by Ryan-Myers**
Some of the functions in 5BF20 seem to be related to the photo printer, but it's not yet clear what it's doing.

I commented out the function declaration for `func_80371D14_8454C4` as it was causing problems with m2c, and the permuter.

### CODE REVIEW

File: src/5BF20.c [hunk 1]
```diff
@@ -78,11 +317,30 @@
...
+
+void func_800C02A0_5D140(s32 arg0) {
+    if (arg0 < 0) {
+        arg0 = 0;
+    }
+    if (arg0 >= 7) {
+        arg0 = 6;
+    }
+    D_800C21B0_5F050->unk50 = arg0;
+    if ((D_800C21B0_5F050->unk64 >> 29) < arg0) {
```
Comment (marijnvdwerf) [inline src/5BF20.c:340]: Have you tested this as a bitfield? Shifting by 29 seems like a _lot_.

File: include/functions.h [hunk 2]
```diff
@@ -110,7 +114,7 @@ void func_80370A48_8441F8(void);
 void func_803713C8_844B78(s32);
 void func_8037172C_844EDC(s32);
 void func_803717E8_844F98(void);
-GObj* func_80371D14_8454C4(void (*gfxFunc)(GObj*), s32 arg1, Sprite* sprite);
+// GObj* func_80371D14_8454C4(void (*gfxFunc)(GObj*), s32 arg1, Sprite* sprite);
```
Comment (ethteck) [inline include/functions.h:117]: I had to uncomment this too for context reasons, but we should add it back and probably soon fix the header ordering so it's not as broken

File: src/5BF20.c [hunk 3]
```diff
@@ -1,56 +1,293 @@
...
-
-#pragma GLOBAL_ASM("asm/nonmatchings/5BF20/func_800BF244_5C0E4.s")
-
-#pragma GLOBAL_ASM("asm/nonmatchings/5BF20/func_800BF37C_5C21C.s")
-
-#pragma GLOBAL_ASM("asm/nonmatchings/5BF20/func_800BF3A8_5C248.s")
-
+// sizeof: 0x3A0?
+typedef struct Unk_D_800C21B0_5F050 {
+    s32 unk0;
```
Comment (ethteck) [inline src/5BF20.c:5]: I prefer an underscore between unk and the offset name (unk_50)

File: src/5BF20.c [hunk 4]
```diff
@@ -78,11 +317,30 @@
...
+
+s32 func_800C0290_5D130(void) {
+    return D_800C21B0_5F050->unk50;
+}
+
+void func_800C02A0_5D140(s32 arg0) {
+    if (arg0 < 0) {
+        arg0 = 0;
+    }
+    if (arg0 >= 7) {
```
Comment (ethteck) [inline src/5BF20.c:336]: `> 6`

**Review by ethteck — CHANGES_REQUESTED** [review]
thanks for all these! a couple minor requests

File: src/5BF20.c (hunk 1 above)
Comment (Ryan-Myers) [reply src/5BF20.c]: Yeah, it works with a bitfield. I've added it now.

File: include/functions.h (hunk 2 above)
Comment (Ryan-Myers) [reply include/functions.h]: Sure, I've added it back in.

File: tools/symbol_addrs.txt [hunk 5]
```diff
@@ -327,6 +327,7 @@ func_800AAF10 = 0x800AAF10;
 gLevelID = 0x800AC0C0;
 gLevelNames = 0x800AC0C4;
 gPhotoCount = 0x800AC0E0;
+get_player_name = 0x800BF498;
```
Comment (ethteck) [inline tools/symbol_addrs.txt:330]: Sorry, this is my bad.. It should probably be getPlayerName.. Or in general, we should all discuss a naming convention scheme

**Pull request merged**